#! /usr/bin/python

PACKAGE = "robot_interface"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("WaypointInterval", double_t, 0, "Desired distance b/w two points calculated by computeCartesianPath function", 0.001, 0.0005, 0.1)

exit(gen.generate(PACKAGE, "robot_interface", "RobotInterface"))
